src/api/spec/support/shared_examples/controllers/image_templates.rb

```diff
@@ -0,0 +1,26 @@
+RSpec.shared_examples 'image templates' do |template, format|
+  context 'without image templates' do
+    before do
+      get :index, format: format
+    end
+
+    it { is_expected.to render_template("webui/image_templates/#{template}") }
+    it { is_expected.to respond_with(:success) }
+    it { expect(assigns(:projects)).to eq([]) }
+  end
+
+  context 'with image templates' do
+    let(:attribute_type) { AttribType.find_by_namespace_and_name!('OBS', 'ImageTemplates') }
+    let(:leap_project) { create(:project, name: 'openSUSE_Leap') }
+    let!(:attrib) { create(:attrib, attrib_type: attribute_type, project: leap_project) }
+    let!(:tumbleweed_project) { create(:project, name: 'openSUSE_Tumbleweed') }
+
+    before do
+      get :index, format: format
+    end
+
+    it { is_expected.to respond_with(:success) }
+    it { expect(assigns(:projects)).to eq([leap_project]) }
+    it { is_expected.to render_template("webui/image_templates/#{template}") }
```

Comment:
very minor point but can you re-order  these three lines so that they match the order in the first context? i think it makes it easier to see what the difference in expectations are between these two contexts.